projects/demo/src/modules/components/dialog/examples/6/index.html

```diff
@@ -1,7 +1,7 @@
 <button tuiButton type="button" size="m" (click)="showDialog()">Show</button>
 <ng-template
     let-observer
-    [tuiDialogOptions]="{ label: 'Declarative directive', size: 's' }"
+    [tuiDialogOptions]="$any({ label: 'Declarative directive', size: 's' })"
```

Comment:
Same here

Reply:
can we leave TODO here for now? and solve it in the next PR?